server/app.ts

```diff
@@ -10,10 +11,12 @@ import ride from './router/ride';
 import vehicle from './router/vehicle';
 import location from './router/location';
 import auth from './router/auth';
+import upload from './router/upload';
 
 const port = process.env.PORT || 3001;
 
 dynamoose.aws.sdk.config.update(config);
+AWS.config.update(config);
```

Comment:
Is this doing the same thing as the line above it? Looks very similar. If so, remove this line and the import for aws-sdk

Reply:
oh yeah I removed it